transpiler/src/main/java/org/jsweet/transpiler/extension/RemoveJavaDependenciesAdapter.java

```diff
@@ -1497,7 +1500,13 @@ public boolean substituteNewClass(NewClassElement newClass) {
 		case "java.lang.Float":
 		case "java.long.Short":
 		case "java.util.Byte":
-			print("new Number(").print(newClass.getArgument(0)).print(").valueOf()");
+		    String argType = newClass.getArgument(0).getType().toString();
+			boolean isCharArgument = "java.lang.Character".equals(argType) || "char".equals(argType);
```

Comment:
I think we could use `Character.class.getName()` to avoid class names in strings. 

I know that there is already plenty in this file but we will enhance from time to time :)

Reply:
Good point! Fixed!
I initially tried using `types().isSameType(..., util().getType(Character.class))` but the transpiler errored out just when it came to `Character.class` wasn't sure why.